src/main/java/org/fogbowcloud/saps/engine/core/scheduler/arrebol/Arrebol.java

```diff
@@ -20,11 +21,13 @@
 
 	public List<JobSubmitted> returnAllJobsSubmitted();
 
-	public JobResponseDTO checkStatusJobById(String jobId) throws GetJobException;
+	public JobResponseDTO checkStatusJobById(String jobId)
+        throws GetJobException, ArrebolConnectException;
 	
 	public List<JobResponseDTO> checkStatusJobByName(String JobName) throws GetJobException;
```

Comment:
Add a fixme reporting that the method is waiting for Arrebol to implement this feature